modules/nodes-nodes-audit-policy-custom.adoc

```diff
@@ -41,11 +46,11 @@ spec:
     profile: Default                    <2>
 ----
 <1> Add one or more groups and specify the profile to use for that group. These custom rules take precedence over the top-level profile field. The custom rules are evaluated from top to bottom, and the first that matches is applied.
-<2> Set to `Default`, `WriteRequestBodies`, `AllRequestBodies`, or `None`. If you do not set this top-level `audit.profile` field, it defaults to the `Default` profile.
+<2> Set to `Default`, `WriteRequestBodies`, or `AllRequestBodies`. If you do not set this top-level profile field, it defaults to the `Default` profile.
 +
 [WARNING]
 ====
-It is not recommended to disable audit logging by using the `None` profile unless you are fully aware of the risks of not logging data that can be beneficial when troubleshooting issues. If you disable audit logging and a support situation arises, you might need to enable audit logging and reproduce the issue in order to troubleshoot properly.
```

Comment:
the deleted line is still relevant, no?

Reply:
we have it already in https://github.com/openshift/openshift-docs/pull/64307/files#diff-9290219f9e18b4b469e81f5f7f90ebacf82ff5e53628126120cb460fc9bca5e8R31-L33 and since we don't recommend setting None in the top-level profile when defining custom rules, I think it makes sense to remove it here no?

Reply:
Yeah we have this warning next to the "None" in the table of profiles and in the procedure where you set the log policy.

I removed it here since it really wasn't relevant if you can't set it to "None" if you want custom rules.

Reply:
oh, yeah, it makes sense